python/examples/criteo_ctr_with_cube/cube/conf/transfer.conf

```diff
@@ -0,0 +1,17 @@
+[default]
```

Comment:
Do you still need this file for example?

Reply:
no need anymore, I remove it.